README.md

```diff
@@ -1,2 +1,86 @@
-# to-do-list
-The to do list app is the app that helps people to track what they have to do in a day. With this app, you can add a new activity with description, remove it and also check it if it is completed.
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## To-do-list-app
+
+- The to do list app is the app that helps people to track what they have to do in a day. With this app,
+  you can add a new activity with description, remove it and also check it if it is completed.
+  There is a lot more of what someone can do with it.
+
+## Built With
+
+- HTML and CSS
+- plain Javascript
+
+## How to use and run this project
+
+> To get a local copy up and running follow these simple example steps.
+
+-Clone this repository with
+
+<code> git clone <https://github.com/BertrandConxy/To-do-list-app.git> </code>
+
+-using your terminal or command line.
+
+-Change to the project directory by entering :
+
+<code>cd To-do-list-app</code>, in the terminal.
+
+-run code to open it in vscode.
+However, there are some project requisites required for this project to run properly
+
+### Prerequisites
+
+- This project makes use of bundler called 'Webpack' which manages all the dependencies and files for this project.
+- To install it, follow this:
+- Locate to the directory of the project
+- In the terminal, <code>npm init -y</code> to initialise the package manager
+- Again, in the terminal, <code> npm install </code> to install everything.
+
+## Project Status
+
+This project is finished and deplyed with Github pages.
+
+## Screenshoots
+
+![Screenshot (116)](https://user-images.githubusercontent.com/90222110/150308929-86440b0b-7f5f-4bad-9aa3-44801c45f0c0.png)
+
+## Live Demo link
+
+<https://bertrandconxy.github.io/To-do-list-app/>
+
+## Issues
+
+Up to now, there are no issues with it.
+
+Here is the link to the Issues tab:
+
+<https://github.com/BertrandConxy/To-do-list-app/issues>
+
+## Authors
+
+👤 **Bertrand Mutangana Ishimwe**
+
+- GitHub: [@BertrandConxy](https://github.com/BertrandConxy)
+- Twitter: [@Bconxy](https://twitter.com/Bconxy)
+- LinkedIn: [Bertrand Mutangana Ishimwe](https://www.linkedin.com/in/bertrand-mutangana-024905220/)
+
+👤 **Daniel Yerimah**
+
+- GitHub: [@yerimah](https://github.com/yerimah)
+- Twitter: [@iamdeewyne](https://twitter.com/iamdeewyne)
+- LinkedIn: [@daniel-yerimah](https://www.linkedin.com/in/daniel-yerimah/)
+
+👤 **Adekunle Ismael**
+
+- GitHub: [@yerimah](https://github.com/ismailakinkunmi)
+- Twitter: [@iamdeewyne](https://twitter.com/AkinkunmiAbiola)
+- LinkedIn: [@daniel-yerimah](https://www.linkedin.com/in/ismail-akinkunmi-adekunle/)
+
```

Comment:
- [ ] You have done a good job crafting a detailed README file. However, this project is a solo project. I think you should modify the README.md file to reflect this.